components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FxaDeviceConstellation.kt

```diff
@@ -206,7 +206,10 @@ class FxaDeviceConstellation(
 
             // NB: at this point, 'constellationState' might have changed.
             // Notify with an immutable, local 'newState' instead.
-            deviceObserverRegistry.notifyObservers { onDevicesUpdate(newState) }
+            deviceObserverRegistry.notifyObservers {
+                logger.info("Notifying observer about constellation updates: $this")
```

Comment:
`this` prints out the class name and the memory address which isn't super useful.